DiscordBot.DataAccess/EventsSheetsService.cs

```diff
@@ -26,6 +31,12 @@ public interface IEventsSheetsService
 
         Task<DiscordEvent> GetEventAsync(int eventKey);
         Task<IEnumerable<DiscordEvent>> ListEventsAsync();
+
+        Task AddResponseOfUser(int eventKey, ulong userId, string responseEmoji);
+        Task ClearResponsesOfUser(int eventKey, ulong userId);
+
+        Task<Dictionary<ulong, IEnumerable<EventResponse>>> GetSignupsByUser(int eventId);
```

Comment:
Why `ulong`? 🙂

Reply:
The Discord API stores and processes user IDs as `ulong`.

Reply:
I see 🙂